superflore/generators/bitbake/run.py

```diff
@@ -76,6 +77,22 @@ def main():
         warn('"{0}" distro detected...'.format(args.ros_distro))
         selected_targets = [args.ros_distro]
         preserve_existing = False
+    # open cached tar file if it exists
+    md5_cache = None
+    sha256_cache = None
+    if args.tar_dir:
+        try:
+            md5_file = open('%s/md5_cache.pickle', 'rb')
+            md5_cache = pickle.load(md5_file)
+            md5_file.close()
+        except IOError:
+            md5_cache = dict()
```

Comment:
Should this be None so it's the same as if the option wasn't passed?

Reply:
I think my line of thought there was _they want to do a cache, so we'll initialize a dictionary to store that option, so we can check later that it's not `None`._

Sorry, this branch was far from ready for review... I was just opening this so you could see it.